### PR TITLE
Edit my contact form

### PR DESCRIPTION
**Pull request by ReemMohamedAbdelfatah**
Edit my Contact-me form with the requirements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -267,24 +267,26 @@ <h1 class="header-languages">Skills</h1><span><img src="icons/RightArrow.png" al
...
       <section class="contactme-section">
         
-        <div class="contactme-form">
+        <form action="https://formspree.io/f/contactme-form" method="post" id="contactme-form" class="contactme-form">
             <h1 class="contactme-header">Contact me</h1>
             <p class="contactme-paragraph">
                 If you have an application you<br> are interested in developing, a <br>
                 feature that you need built or a <br> 
                 project that needs coding. I’d<br>love to help with it 
             </p>
```
Comment (RolandM99) [inline index.html]: - [ ] Kindly, make sure that you remove these codes from your `<form>` tag. your `form` tag should only wrap the input for the name and email, and also the textarea and the submit button.

File: index.html [hunk 2]
```diff
@@ -267,24 +267,26 @@ <h1 class="header-languages">Skills</h1><span><img src="icons/RightArrow.png" al
...
-                <li ><div class="form-textBox smallTxtbox"><span class="TextBox-personal-style">Yeremias</span></div></li>
-                <li ><div class="form-textBox smallTxtbox"><span class="TextBox-personal-style">natajayanj14@gmail.com</span></div></li>
-                <li ><div class="bigTextBox"><span class="TextBox-info-style">Write your message here</span></div></li>
-            </ul>
+            
+                <input type="text" id="sender-name" name="name" class="form-textBox smallTxtbox" placeholder="Yeremias">
+               
+                <input type="email" id="mail" name="user_email" class="form-textBox smallTxtbox" placeholder="natajayanj14@gmail.com">
+                
+                <textarea class="form-textBox bigTextBox">Write your message here</textarea>
```
Comment (RolandM99) [inline index.html]: - [ ] Your name input should have a `maxlength="30"` and your textarea a `maxlength="500"` attributes.
- [ ] All input fields should have the `required` attribute to activate the validation on submission.
- [ ] For now, when we submit the form, the error below is raised. kindly, make sure that you fix it by creating your account at [formspree service](https://formspree.io/html). 
image

Then, you'll generate your own key_id which you'll use for the action value.
your action value should be similar to this: https://formspree.io/f/{form_id}

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @ReemMohamedAbdelfatah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @ReemMohamedAbdelfatah,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ReemMohamedAbdelfatah** [comment]
I happily finished my Portfolio mobile version :)

**Pull request merged**